README.md

```diff
@@ -67,5 +67,12 @@ For the last spec `05_song_form_spec.rb` you need to implement the following fea
 
 Think about the custom writer or writers you may need to write to make these features work. 
 
+### Slugs
+
+You are going to need to create some slugs in this lab. A slug is used to create a name that is not acceptable as a URL for various reasons (special characters, spaces, etc). This is great because instead of having a route like `/songs/1`, you can have a route `/songs/hotline-bling` which is a much more descriptive route name.
```

Comment:
So while slugs are about the url validation issues, I would express this starting with the conflict a bit more:

 1. Having urls like `/songs/1` sucks. When you email that to someone, they have no idea what song that is without clicking on the link. it'd be better if it was `/songs/hot line bling`
 2. Can't have spaces in urls
 3. So we need to convert spaces to `-`

That is a slug